### PR TITLE
Remove command prompt from Windows example

### PR DESCRIPTION
**Pull request by merkata**
According to the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt), the command prompt should not be included in the examples. Removing the
Windows command prompt from the jsonpath example.

Signed-off-by: Mariyan Dimitrov <mariyan.dimitrov@gmail.com>

### CODE REVIEW

File: content/en/docs/reference/kubectl/jsonpath.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ kubectl get pods -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.status.st
 On Windows, you must _double_ quote any JSONPath template that contains spaces (not single quote as shown above for bash). This in turn means that you must use a single quote or escaped double quote around any literals in the template. For example:
 
 ```cmd
-C:\> kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{'\t'}{.status.startTime}{'\n'}{end}"
-C:\> kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{\"\t\"}{.status.startTime}{\"\n\"}{end}"
+kubectl get pods -o=jsonpath="{range .items[*]}{.metadata.name}{'\t'}{.status.startTime}{'\n'}{end}"
```
Comment (kbhawkey) [inline content/en/docs/reference/kubectl/jsonpath.md]: Hello @merkata . If I recall correctly, lines 94 - 95 were formatted explicitly for `Windows` (hence the C:\ prompt).
Did the prompt help indicate that the command is targeted for Windows users or does the description on line 92 provide enough information?

Comment (merkata) [reply content/en/docs/reference/kubectl/jsonpath.md]: This is due to #18853 and the comment https://github.com/kubernetes/website/pull/18853#discussion_r372823426 from @sftim

Comment (kbhawkey) [reply content/en/docs/reference/kubectl/jsonpath.md]: @merkata , Yes, the style guide recommends to exclude a command prompt.
To emphasize that this section applies to Windows users, I am wondering if lines 92 - 96 should be enclosed
in a note shortcode?
```
{{< note >}}

{{< /note >}}
```

Comment (sftim) [reply content/en/docs/reference/kubectl/jsonpath.md]: Is it possible to write one sample command that works on both POSIX / Bourne shell and Windows command shell (maybe PowerShell too?)

Comment (merkata) [reply content/en/docs/reference/kubectl/jsonpath.md]: @kbhawkey good point, I've pushed the change, thanks!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-18906--kubernetes-io-master-staging.netlify.com/docs/reference/kubectl/jsonpath/

**Pull request merged**